site/content/pages/organizing/04-Running your event/07-video-streaming.txt

```diff
@@ -0,0 +1,17 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Video - recording and streaming
+---
+
+Whether or not you're able to livestream, it's important to record all the talks. This is invaluable for your speakers, and it's great for the community. Your audiovisual company should be able to provide a camera or cameras and record the feed off the board into a computer.
```

Comment:
@bridgetkromhout let's leave it as-is for now and I'll shoot an email to the list about video stuff.